docs/advanced.rst

```diff
@@ -230,6 +230,51 @@ example.
 
 TCP doesn't support multicast scenarios, so you must plan carefully your network architecture.
 
+
+Using TLS (Transport Layer Security) over TCP
+---------------------------------------------
+
+Fast-RTPS allows configuring a TCP Transport to use TLS. We must configure both **TCP Server** and **TCP Client**
```

Comment:
Break the line. And second line is a little repetition. Also `We must set TCP server and TCP Client propertly`

Reply:
What about "Fast-RTPS allows configuring a TCP Transport to use TLS by setting up **TCP Server** and **TCP Client** properly."?